pages/benefit-plans.tsx

```diff
@@ -74,12 +75,27 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
 
   const prisma = new PrismaClient();
 
-  const benefitPlans: BenefitPlan[] = await prisma.benefitPlan.findMany({
+  const dbBenefitPlans: BenefitPlan[] = await prisma.benefitPlan.findMany({
     where: {
       organizationId: token.organizationId,
     },
   });
 
+  let benefitPlans = null;
+  if (dbBenefitPlans) {
```

Comment:
Do you need this if check? What's the default return from `findMany`, isn't it `[]`?

Reply:
removed unneeded if